### ISSUE TITLE
New states "reigned_by" field

### ISSUE THREAD

**Issue by RiLeone** [RiLeone/warbot#5]
To have longer and potentially more interesting histories, add a new property to the states called "reigned_by". The idea is that each state somehow "survives" forever. Also, like this states should then only be able to "attack" neighbouring regions (i.e. states which might or might not be autonomous). A controlled region should also have the option to rebel against the reigning state and try to regain independence.

**Comment by SGE-MagHead** [comment]
It would be interesting also to implement on the rebellion style that there is a certain percentage that a very big state (coquered a lot of other states) brakes up in a moltitude of small one (USSR style)

**Comment by mafilipp** [comment]
Are both very good ideas, but I think that for the design of the whole game we should have a meeting together and decide all the rules. If everyone implements something in the end imo it will become a big mess

**Comment by SGE-MagHead** [comment]
I disagree with becoming a mess, if we keep the contributing rules we should minimize the risk of conflicting modules. We should try to have it in a ROS manner where every module/extension can be run/enable on the fly. I think every body should also have an experimental branch where to test stuff with respect to the current master to sort out conflicts
